src/Locale.cpp

```diff
@@ -825,19 +830,16 @@ ShowBug(const char* string)
 	message << "Error: " << string << "\n";
 
 	if (value == 0) {
-		// Make an e-mail to myself. :D
-
-		BString cmdstring("/boot/beos/apps/BeMail mailto:support@capitalbe.com ");
-		cmdstring << "-subject 'CapitalBe Bug Report' -body '" << message << "'";
-		cmdstring << " &";
-		system(cmdstring.String());
-	} else if (value == 1) {
 		// Generate a text file of the bug on the Desktop
 		BString filename("/boot/home/Desktop/CapitalBe Bug Report ");
 		filename << real_time_clock() << ".txt";
 		BFile file(filename.String(), B_READ_WRITE | B_CREATE_FILE | B_ERASE_FILE);
 		file.Write(message.String(), message.Length());
 		file.Unset();
+
+		// Open GitHub in browser
+		char* argv[2] = {(char*)"https://github.com/HaikuArchives/CapitalBe/issues/", NULL};
+		be_roster->Launch("text/html", 1, argv);
```

Comment:
BURL("https://github.com/HaikuArchives/CapitalBe/issues/").OpenWithPreferredApplication() would be preferred.

Reply:
@korli, is there another reason to do it like this, besides that's just what BURL was made for?
Just asking for other projects doing similar things...

Reply:
This is what 'open' uses,  and in fine use the correct mime type application/x-vnd.Be.URL.https

Reply:
Thanks!